LayoutTests/imported/w3c/web-platform-tests/xhr/formdata/constructor-submitter.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
```

Comment:
In WebKit, you need to include the generated constructor-submitter-expected.txt as part of your patch (it gets generated when you run this test locally).